common/cfme-install-appliance-as-vm.adoc

```diff
@@ -0,0 +1,23 @@
+// Using virt-manager command, we will install the CF appliance. Ensure the cfme.qcow2 file is placed in /var/lib/libvirt/images/ directory and make sure the security context of the file is correct. Use 4 vcpus and 8192 MiB of RAM for the appliance. Add a second network interface of virto type. 
+
+. Log in as `root` to the virtual machine host.
+. Copy the appliance to `/var/lib/libvirt/images/`.
+. Run `virt-manager`. Doing so will launch the Virtual Machine Manager.
+. Enter a name for the VM in the *Name* field; for example, use `my-cfme`. Select *Import existing disk image* and click *Forward*.
++
+image:virt-manager-install-step1.png[]
+. Click *Browse* to select the copy of the appliance stored in `/var/lib/libvirt/images/`.
++
+image:virt-manager-install-step2.png[]
++
+Select *Linux* from the *OS type* drop-down. For *Version*, select *Red Hat Enterprise Linux 7 or later*. Click *Forward*.
+. Configure the appliance with 4 CPUs and 8192MiB or memory. Select *Customize configuration before install* then click *Finish*.
++
+image:virt-manager-install-step3.png[]
+. Add a second network interface for the VM. Select *virtio* as its *Device Model*.
++
+image:virt-manager-install-step4.png[]
+. Configure the VM with two additional virtual disks. One will be used for the internal database (xref:configuring_a_database[]), while the other will be used for mounting images and SmartState analysis.
++
+image:virt-manager-install-step5.png[]
+. Click *Finish* to launch the VM. 
```

Comment:
VM should be spelled out as virtual machine throughout (https://mojo.redhat.com/docs/DOC-1076002).